identity/client/src/pages/users/detail/authorization/List.tsx

```diff
@@ -34,8 +63,49 @@ const List: FC<AuthorizationsListProps> = ({ user, loadingUser }) => {
   const areAuthorizationsEmpty =
     !authorizations || authorizations.items.length === 0;
 
-  const showAuthorizationDetails = (authorization: Authorization) =>
-    console.log(authorization);
+  const headers = [{ header: t("ResourceType"), key: "resourceType" }];
+
+  const rows = authorizations == null ? [] : authorizations.items;
+  // Create a lookup map for rows by ID
+  const dataMap = rows.reduce(
+    (map, item) => {
+      if (typeof item.key === "string") {
+        map[item.key] = item;
+      }
```

Comment:
I added this if but this breaks the behavior, which was already broken.
There's no `key` property coming in the payload so the table is not properly rendered and it always render the last item in the list

https://github.com/user-attachments/assets/efd65925-cbac-462a-a888-82748d863e06